media/css/m24/flag.scss

```diff
@@ -20,17 +20,81 @@
 
 .m24-c-flag-subtitle {
     font-size: $text-title-md;
-    text-wrap: balance;
-    text-wrap-style: balance;
     margin-bottom: $spacer-xl;
 }
 
+.m24-c-flag-cta {
+    margin-bottom: 0;
+}
+
 .m24-c-flag-media {
-    display: block;
-    margin-bottom: 16px;
+    --delay: 1s; // leave some space to read before introducing movement
+    --wave-h: 8px;
 
-    img {
+    svg {
+        transform: translateZ(0);
         width: 100px;
+        height: auto;
+        fill: $m24-color-green; // zilla green
+    }
+}
+
+.m24-c-flag-button {
+    $font-size: 0.75;
+    all: unset;
```

Comment:
This removed the focus ring. Can you add it back please? Ideally with [the native focus ring ](https://css-tricks.com/copy-the-browsers-native-focus-styles/). But if that is finiky any kind of outline would be a good addition.